python/tests/test_base_install/test_graphql/test_rolling_expanding.py

```diff
@@ -2041,3 +2041,242 @@ def test_wrong_window():
     """
     queries_and_exceptions.append((query, mismatch_exception))
     run_group_graphql_error_test(queries_and_exceptions, graph)
+
+def test_alignment():
+    g = Graph()
```

Comment:
These tests all look great! Can you just add a couple weird ones like "2 weeks" "3 days"  "2 weeks, 3 days" etc so we have some durations that non singular units 

Sorry meant to put this on the file above when you were doing the rolling/expanding in python tests

Reply:
The implementation all looks good for this, just so we have the tests available in case we change the implementation later